tests/functional/forklift/test_legacy.py

```diff
@@ -48,3 +56,61 @@ def test_remove_doc_upload(webtest):
         "410 Uploading documentation is no longer supported, we recommend "
         "using https://readthedocs.org/."
     )
+
+
+def test_file_upload(webtest):
+    user = UserFactory.create(
+        password=(  # 'password'
+            "$argon2id$v=19$m=1024,t=6,p=6$EiLE2Nsbo9S6N+acs/beGw$ccyZDCZstr1/+Y/1s3BVZ"
+            "HOJaqfBroT0JCieHug281c"
+        )
+    )
+    EmailFactory.create(user=user, verified=True)
```

Comment:
We have a UserFactory trait now, would that accomplish your needs?
```suggestion
    user = UserFactory.create(with_verified_primary_email=True)
```

What purpose does setting a hashed password value here? Could we supply a string instead, or does some other mechanic prevent this?

Reply:
I was having an issue where the factory was not actually making this a valid 'enabled' password, and the user account was not usable as a result. Maybe this should be a default value? I think we can definitely use the trait though.

Reply:
Yeah this is why: https://github.com/pypi/warehouse/blob/ccff64920db7965078cf1fdb50f028e640328887/tests/common/db/accounts.py#L39

Causes this to fail: https://github.com/pypi/warehouse/blob/ccff64920db7965078cf1fdb50f028e640328887/warehouse/accounts/services.py#L342

Reply:
Yeah, the default password is `"!"` which is unix-y for disabled. We could probably change our default and see what happens!